doc/ideas/fr_technical_overview.md

```diff
@@ -0,0 +1,137 @@
+Gameplay           | GM7        | New Unit: transport waggon on land to slowly transport ships over land
+Gameplay           | GM8        | villagers need sleep at night, die at work (New Building: Caffeine industry)
+Gameplay           | GM9        | GAIA and terrain interaction, e.g. flock of bird rises up from tree when tree is felled or many people approach, hearable sound indicator, players hear and see birds when someone comes to their woodline at home. triggered rocks falling of cliffs.
+Gameplay           | GM10       | Relics have special abilities (AOM), bonuses for attack, range. generated randomly. maybe also rolling relics with ranged attributes to bring into battle.
+Gameplay           | GM11       | Bring Kings/Queens into battle for special abilities within some tiles radius.
+Gameplay           | GM12       | Building produces in loop (AOM) as long as resources and pop space are available. units and farms. indicator over buildings or in the GUI for loop production. Toggle reseeding on/off.
+Gameplay           | GM13       | Mills can produce sheep, cows, turkeys
+Gameplay           | GM14       | Researching faith in third age allows to convert units, before monk can heal and carry relics.
+Gameplay           | GM15       | Buildings can be deconstructed to recover materials, slower than destroying. leave a salvage pile behind. salvage dimishes per time. resources can be harvested by villagers on a higher rate.
+Gameplay           | GM16       | Buildings can be captured when damaged to 20% or less. Player who repairs over 20% owns the building. New units can be produced out of this captured building from 50% on.
+Gameplay           | GM17       | Attack/defense bonuses for some unit formations. Dependent on minimum amount of units in this formation. New marching formation with speed bonus but defense malus. Cavalry flanking.
+Gameplay           | GM18       | Researches only affect newly produced units. Old ones need to be manually upgraded/retrained at production building.
+Gameplay           | GM19       | Scale wonder cost to max pop, number of players and available resources.
+Gameplay           | GM20       | a new 5th "free trade" age where allies share resources and give each other massive battles.
+Gameplay           | GM21       | New unit: Lance Cavalry, run over units in defending formations
+Gameplay           | GM22       | Researches can be globally and locally applied. e.g. upgrade just units garrisoned in the barracks for less research cost; e.g. you need 3 pikemen for the battle now, but you don't want to produce them in the future in masses (see GM18)
+Gameplay           | GM23       | Units can level up by gaining experience doing what they normally do, fighting in battle against their conter unit will give more XP against that unit style. Villagers chopping a forest nothing map might be a real pro at chopping trees in the end of the game.
```

Comment:
conter --> counter (unit)
forest nothing --> Forest Nothing
in the end --> at the end

Reply:
done